upgrade/pack_builder/Input.hpp

```diff
@@ -6,6 +6,15 @@
 
 namespace application
 {
+    struct TargetNameTooLongException
```

Comment:
Why a struct? You could also hide the data members and implement the what() method.

Reply:
All the other exception types used in UpgradePackBuilderFacade::Build were defined as such structs with public fields. Adapted them all.